include/geom/elem.h

```diff
@@ -1065,6 +1065,21 @@ class Elem : public ReferenceCountedObject<Elem>,
    */
   virtual bool close_to_point(const Point & p, Real tol) const;
 
+  /**
+   * \returns \p true if edge \p i is positively oriented. An edge is
+   * positively oriented iff its first vertex (i.e. zeroth node) is
+   * lexicographically greater than its second vertex (i.e. first node).
+   */
+  bool edge_orientation(const unsigned int i) const;
```

Comment:
I'd prefer to name these functions `positive_edge_orientation()` and `positive_face_orientation()`, respectively, so that it's more clear what the `true`/`false` return value is referring to.

Reply:
Done.